CMake/FindSqlite3.cmake

```diff
@@ -1,2 +1,15 @@
+add_definitions(
+  -DSQLITE_ENABLE_COLUMN_METADATA
+  -DSQLITE_ENABLE_API_ARMOR
+  # Functionality disables
+  -DSQLITE_DISABLE_LFS
```

Comment:
These generate more unused variable warnings within the SQLite code, harmless, but annoying. :/